src/info/langs/mod.rs

```diff
@@ -9,14 +9,14 @@ use strum::IntoEnumIterator;
 
 pub mod language;
 
-pub fn get_main_language(loc_by_language: &[(Language, usize)]) -> Language {
```

Comment:
A bit of refactoring is inevitable to make things make more sense when using `gengo` over `tokei`. Stumbled on this. Don't mean to be harsh, but is a function for the sole purpose of indexing into `[0]` really necessary? Especially now that newer versions of Rust have `.first()` and `.get(0)` to *safely* index into collections.

Reply:
Sure, go ahead. 

But for an easier review, we can keep the refactoring/cleanup in a separate PR, so we can focus on the current subject and don't lose track of the important changes.

Reply:
Refactored it to handle a OOB panic that was introduced (see https://github.com/o2sh/onefetch/pull/1305/commits/82da923be01d27b7eed51e0835e7e239582ba512 for more info).